src/Autofac.Integration.Wcf/DefaultServiceImplementationDataProvider.cs

```diff
@@ -80,7 +80,7 @@ public virtual ServiceImplementationData GetServiceImplementationData(string val
                 }
             }
 
-            if (serviceRegistration == null)
+            if (serviceRegistration == null || serviceRegistration == default)
```

Comment:
I feel like the null check is unnecessary, due to serviceRegistration being a non-nullable struct. I left it in there to be sure.

Reply:
I think it's fine. I think you're correct, though, that it'll never be hit.